tools/ClassGen/InstrGen.cpp

```diff
@@ -621,6 +621,11 @@ int main(int argc, char **argv) {
       .autoVerify(VerifyKind::NoVerify)
       .autoIRGen();
 
+  BB.newInstr("Touch")
+      .addOperand("Dest", OperandKind::Out)
+      .dataParallel()
```

Comment:
We can skip this, right? Seems a little misleading/weird to mark essentially a nop as data parallel.

Reply:
We cannot skip this. I'm not 100% sure why but Instructions that only have an output (e.g. splat) must be data parallel or they don't work.